packages/app/features/dashboard/profile/edit/index.tsx

```diff
@@ -20,46 +16,61 @@ import { userAtom } from "app/state/user";
 import { FormInputWithIcon } from "app/ui/inputs/FormInputWithIcon";
 import { ScrollView } from "app/design-system/ScrollView";
 import { useRouter } from "solito/router";
+import { ChangeAvatarImg, EditProfileForm } from "app/types";
+import * as assert from "assert";
+import { editProfileFormSchema } from "app/validation";
+import useUploadFileToNFTStorage from "app/hooks/useUploadFileToNFTStorage";
+import { ipfsProtocol } from "app/constants/constants";
+import { toast } from "app/utils/toast";
 
 export default function EditProfileScreen() {
   const [user, setUser] = useRecoilState(userAtom);
+  assert.ok(user, "Unauthorized access on EditProfileScreen");
```

Comment:
Using `!` is an anti-pattern (it does not provide runtime checks, and here we would need to reassign user for typescript to correctly remove null from possible types) using `?` every time is unnecessary because we known that user is not null. So this is what i found